app/experimenter/experiments/tests/test_models.py

```diff
@@ -1105,6 +1105,16 @@ def test_is_ready_to_launch_is_false_without_conditional_review(self):
 
         self.assertFalse(experiment.is_ready_to_launch)
 
+    def test_completed_results_returns_true_if_any_results(self):
+        experiment = ExperimentFactory.create(
+            results_initial="The results here were great."
+        )
+        self.assertTrue(experiment.completed_results)
+
+    def test_completed_results_returns_false_if_none(self):
+        experiment = ExperimentFactory.create()
```

Comment:
I'd explicitly set those fields to None here, I've done this kind of thing in the past and then we change the way the factory behaves and this will break.